packages/server/graphql/mutations/removeTeamMember.ts

```diff
@@ -41,7 +41,7 @@ export default {
     const {user, notificationId, archivedTaskIds, reassignedTaskIds} = res
 
     const teamMembers = await dataLoader.get('teamMembersByTeamId').load(teamId)
-    const {tms} = user
+    const {tms} = user!
```

Comment:
Maybe check if user exists, feels not very obvious by looking at lines above
```
const res = await removeTeamMember(teamMemberId, {evictorUserId}, dataLoader)
const {user, notificationId, archivedTaskIds, reassignedTaskIds} = res
```

Reply:
done